DQMOffline/Trigger/python/ObjMonitor_cfi.py

```diff
@@ -0,0 +1,75 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.objMonitoring_cfi import objMonitoring
+
+hltobjmonitoring = objMonitoring.clone()
+hltobjmonitoring.FolderName = cms.string('HLT/GENERIC/')
+hltobjmonitoring.doMETHistos = cms.bool(True)
+hltobjmonitoring.histoPSet.metPSet = cms.PSet(
```

Comment:
so I did but only for the histograms that make sense, for which turn-on vs variable makes sense. I think for the others it is useful to have the distributions with and without the filter but not mandatory to have fine binning because no expected turn-on.....